app/assets/stylesheets/components/_share-buttons.scss

```diff
@@ -0,0 +1,51 @@
+.share-buttons {
+  text-align: center;
+  padding-top: $base-spacing * 2;
+
+  h4 {
+    font-weight: $font-weight-normal;
+    color: $medium-gray-color;
+  }
+
+  a {
+    display: inline-block;
+    color: $white-color;
+    border: none;
+    padding: 0;
+    width: 3em;
+    height: 3em;
+    line-height: 3em;
+    opacity: .8;
+    margin: 0 ($small-spacing/2);
+    @include transition(opacity $animation-duration $base-timing);
+    box-shadow: 0 2px 0 0 rgba(0,0,0,0.2);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.2)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space